Polygon_repair_2/include/CGAL/Polygon_repair_2/Polygon_repair_2.h

```diff
@@ -82,6 +82,47 @@ class Polygon_repair_2 {
   typedef CGAL::Constrained_Delaunay_triangulation_2<Kernel, Triangulation_data_structure, Tag> Constrained_Delaunay_triangulation;
   typedef Triangulation_with_odd_even_constraints_2<Constrained_Delaunay_triangulation> Triangulation;
 
+  struct Polygon_compare {
+    bool operator()(const Polygon_2<Kernel, PolygonContainer>& pa, const Polygon_2<Kernel, PolygonContainer>& pb) const {
+      typename Polygon_2<Kernel, PolygonContainer>::Vertex_iterator va = pa.vertices_begin();
```

Comment:
Make a typedef instead of repeating complex types for reasons of readability.

Reply:
> Make a typedef instead of repeating complex types for reasons of readability.

In modern code, [C++ Code Guidelines recommend](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#t43-prefer-using-over-typedef-for-defining-aliases) type aliases (`using Type = ...`) instead of typedefs. As Polygon repair is a new package, all typedefs could be converted to aliases.